### PR TITLE
Add tests

### LINKED ISSUE #63: Add Unit Test for LogicManager

### PR DESCRIPTION
**Pull request by mazx4960**
# Changes
- Refactor duplicate code for commands
- Unit Test for LogicManager
- Unit Test for Item commands

# Related Issues/ PRs
- Fixes #63

### CODE REVIEW

File: src/main/java/seedu/ibook/logic/parser/IBookParser.java [hunk 1]
```diff
@@ -87,21 +84,12 @@ public Command parseCommand(String userInput) throws ParseException {
...
-
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
         case ExpiredCommand.COMMAND_WORD:
             return new ExpiredCommand();
 
-        case RemindCommand.COMMAND_WORD:
-            return new RemindCommandParser().parse(arguments);
-
```
Comment (1001mei) [inline src/main/java/seedu/ibook/logic/parser/IBookParser.java:104]: These lines should be preserved also.

**Review by 1001mei — COMMENTED** [review]
I like how you group related item commands and product commands to new packages!

File: src/main/java/seedu/ibook/logic/parser/IBookParser.java [hunk 2]
```diff
@@ -87,21 +84,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case ClearCommand.COMMAND_WORD:
             return new ClearCommand();
 
-        case UndoCommand.COMMAND_WORD:
-            return new UndoCommand();
-
-        case RedoCommand.COMMAND_WORD:
-            return new RedoCommand();
-
```
[the remark this answers is not in the document]
Comment (mazx4960) [reply src/main/java/seedu/ibook/logic/parser/IBookParser.java]: Thanks for spotting this!

**Review by 1001mei — APPROVED** [review]
LGTM.

**Pull request merged**